shaders/src/surface_main.vs

```diff
@@ -246,8 +246,8 @@ void main() {
     position.y = -position.y;
 #endif
 
-#if !defined(TARGET_VULKAN_ENVIRONMENT) && !defined(TARGET_METAL_ENVIRONMENT)
-    // This is not needed in Vulkan or Metal because clipControl is always (1, 0)
+#if !defined(TARGET_VULKAN_ENVIRONMENT) && !defined(TARGET_METAL_ENVIRONMENT) && !defined(TARGET_WEBGPU_ENVIRONMENT)
```

Comment:
recommend we delay adding this until we understand webgpu's clip control feature better (since it's not always available?).